parm/archive/master_gefs.yaml.j2

```diff
@@ -0,0 +1,12 @@
+# Set variables/lists needed to parse the gefs templates
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+
+datasets:
+{% filter indent(width=4) %}
+    {% include "gefs_extracted_atmos.yaml.j2" %}
+    {% include "gefs_extracted_ocean.yaml.j2" %}
+    {% include "gefs_extracted_ice.yaml.j2" %}
+    {% include "gefs_extracted_wave.yaml.j2" %}
```

Comment:
Why the `extracted`?  Is this intended to be run only when running with `DO_EXTRACTVARS=YES`?  If not, then I suggest dropping the `_extracted`.

Reply:
This was my request because many of the files are only generate when extract variables is on. The idea is to add templates for normal GEFS runs later.

Reply:
OK, sounds good.  Resolving.